viewportwidth.js

```diff
@@ -1,18 +1,12 @@
-/*! viewportwidth.js 0.1.1 | Author: Brett Jankord, 2012 | License: MIT */
-/* https://github.com/bjankord/viewportwidth.js */
-var getViewportWidth = function(){
+module.exports = function() {
+  "use strict";
+  var isWebKit = 'WebkitAppearance' in document.documentElement.style;

```

Comment:
I like this method for detecting webkit better then window.webkitConvertPointFromNodeToPage